test/partitioning.jl

```diff
@@ -196,7 +196,20 @@
   end
 
   @testset "SpatialPredicatePartitioner" begin
-    # TODO
+    g = RegularGrid{Float64}(10,10)
+    
+    # check if each partition has only 1 point
```

Comment:
The test that follows does not check what the comment says. Please make sure that you check the size of each subset in the partition. I showed you before how to iterate over partitions:

```julia
for d in partition(g, sp)
  @test npoints(d) == 1
end
```

Reply:
Marking as solved